config/locales/en.yml

```diff
@@ -194,7 +194,7 @@ en:
       publication_date: 'For example, %{date}'
       benefits: 'List any financial benefits, job training, <span aria-label="Continuing Professional Development">CPD</span>, <span aria-label="Special Educational Needs">SEN</span> allowances, <span aria-label="Teaching and Learning Responsibility">TLR</span> payments, etc.'
       contact_email: 'This email address will be shown publicly as a point of contact'
-      application_link: 'This is the url linking the job to your existing application page'
+      application_link: 'Enter the URL for the webpage that gives details of this job and how to apply. This should be a page on your school’s or trust’s website.'
```

Comment:
I think gramatically, this should be `school or trust's`.

Reply:
Thanks Michael has just confirmed it. Let me update it now.

Reply:
Grammatical mistake corrected